scripts/generate_cli_api.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client CLI API docs generation."""
```

Comment:
We don't need to do this now, but if you agree we can create an issue to add something similar to what we have for the Openapi specs, with a parameter you can instruct the script to copy the file to the correct location (`docs.reana.io/docs/reference/reana-client-cli-api/index.md`), see [here](https://github.com/reanahub/reana-workflow-controller/blob/8ba38e9b44621f58c63da1fd4403f5b4065910fb/scripts/generate_openapi_spec.py#L99-L100) how we do it in RWC.

Reply:
https://github.com/reanahub/reana-client/issues/410